### DISCUSSION TITLE
Ability to make policies optional

### DISCUSSION THREAD

**Discussion by simply-emily** [CareTogether/CareTogetherCMS#256] [Ideas]
Our organization needs the ability to make some policies optional. An example is the RIL, which is not required for children in the home who turn 18 (so needs to be made optional so that a HF can still show as approved).

**Comment by LarsKemmann** [comment]
Would it be better, rather than making requirements _optional_, to have an option to provide an _exemption_ for a requirement (with a note explaining the reason for the requirement exemption)? You could right-click on a requirement and select "Exempt this requirement", then need to provide a note explaining why. That note could then be shown along with the "completed" requirement, using a different icon than ✅ to show that it was an exemption.

Or is the concept of _optional_ requirements really something that will be useful in general? If we can get some other examples of where an "optional" requirement would be useful - as opposed to having an "exemption" option - that would help in making the decision here.

**Reply by simply-emily** [reply]
Yes-- I like the concept of an exemption with a required note better. Excellent idea!
